.circleci/config.yml

```diff
@@ -22,27 +22,28 @@ jobs:
       # build.
       - image: "nixorg/nix:circleci"
 
-    resource_class: "xlarge"
-
     environment:
       # Specify a revision of NixOS/nixpkgs to run against.  This essentially
       # pins the majority of the software involved in the build.  This
       # revision is selected arbitrarily.  It's somewhat current as of the
       # time of this comment.  We can bump it to a newer version when that
       # makes sense.  Meanwhile, the platform won't shift around beneath us
       # unexpectedly.
-      NIXPKGS_REV: "8bf142e001b6876b021c8ee90c2c7cec385fe8e9"
+      NIXPKGS_REV: "5de44c15758465f8ddf84d541ba300b48e56eda4"
```

Comment:
PrivateStorageio has since moved to fd8a7fd07da0f3fc0e27575891f45c2f88e5dd44.  I think we might as well update this to match now.

Reply:
I've bumped it to `92609f3d9bc3acffbdbe54fa1c591a885612aa73` to match https://whetstone.privatestorage.io/privatestorage/PrivateStorageio/-/merge_requests/193 on the presumption that will be merged soon. (Though the impact of this is minimal, as I think we only use it to call `pkgs.fetchFromGitHub` in `nix/challenge-bypass-ristretto-repo.nix`)

Reply:
I also bumped the version in `nix/sources.json, which is also unused, for consistency.